crocks/Pred.js

```diff
@@ -37,7 +40,7 @@ function Pred(runWith) {
       throw new TypeError('Pred.concat: Pred required')
     }
 
-    return Pred(x => runWith(x) && m.runWith(x))
+    return Pred(x => !!runWith(x) && !!m.runWith(x))
```

Comment:
what is this locking down?  lack of errors will make this part confusing to debug [e.g. if a curried function returned is going to always return true]